lib/dockerregistry/types.go

```diff
@@ -39,12 +39,18 @@ type Errors []Error
 type Error struct {
 	Context string
 	Error   error
+	Info    string
```

Comment:
Are we using this?

Reply:
The plan was to use it as a field for extra comments that aren't included in the Error or Context field. For example, telling a user to remove the following images which are too large or something.

Reply:
OK, but if we're not using it in this PR then I suggest not adding it until we actually do need it.